app/models/rental.rb

```diff
@@ -0,0 +1,18 @@
+class Rental < ApplicationRecord
+  belongs_to :video
+  belongs_to :customer
+
+  validate :inventory_available, on: :create
```

Comment:
This could have been replaced with:

```suggestion
   validates :available_inventory, numericality: { greater_than: 0 }
```